int16 x
int16 y
int16 width
int16 height
---
sensor_msgs/Image image
sensor_msgs/Image mask

